### DISCUSSION TITLE
Is Fluent UI Blazor usable in MAUI/Blazor hybrids?

### DISCUSSION THREAD

**Discussion by learntocode66** [microsoft/fluentui-blazor#474] [Q&A] [answered]
Is Fluent UI Blazor usable in MAUI/Blazor hybrids?  Thanks!

**Comment by vnbaaij** [comment, marked as answer]
Yes, it is. See the readme at https://github.com/microsoft/fluentui-blazor#blazor-hybrid. 

There are also some (closed) issues with more information which can be surfaced by using the search function.

**Reply by learntocode66** [reply]
Thank you sir for your quick answer!

**Reply by vnbaaij** [reply]
You're welcome. Please mark it as answered as well. Will help other finding the good information quicker!

**Reply by learntocode66** [reply]
I'm sorry. How can I mark a comment as the answer?
![image](https://github.com/microsoft/fluentui-blazor/assets/137069973/1d53b634-fd3b-4a83-8e14-75b8863d2dfd)
I'm new to GitHub. The ui seems to be a little tricky.

**Reply by vnbaaij** [reply]
The category needed to be changed to Q&A 
<img width="166" alt="image" src="https://github.com/microsoft/fluentui-blazor/assets/1761079/43581f46-76eb-420b-b059-7478214fad3e">
Now you should see a 'Mark as answer' button
